core/extensions/impl/storage_extension.cpp

```diff
@@ -132,11 +132,12 @@ namespace kagome::extensions {
     auto [value_ptr, value_size] = runtime::WasmResult(value_out);
 
     auto key = memory_->loadN(key_ptr, key_size);
+    boost::optional<uint32_t> res{boost::none};
```

Comment:
It initializes with ```none``` in ctor by def, i suppose.

Reply:
yes, but making it explicit improves readability I suppose